discovery-infra/tests/test_kube_api.py

```diff
@@ -213,9 +213,9 @@ def kube_api_test_prepare_late_binding_cluster(kube_api_context, cluster_config:
     agent_cluster_install.create(
         cluster_deployment_ref=cluster_deployment.ref,
         image_set_ref=deploy_image_set(cluster_name, kube_api_context),
-        cluster_cidr=cluster_config.cluster_network_cidr,
-        host_prefix=cluster_config.cluster_network_host_prefix,
-        service_network=cluster_config.service_network_cidr,
+        cluster_cidr=cluster_config.cluster_networks[0].cidr,
+        host_prefix=cluster_config.cluster_networks[0].host_prefix,
+        service_network=cluster_config.service_networks[0].cidr,
```

Comment:
I think it is better to add getters on `cluster_config.py` and use them on all the other places:
e.g. 

```python
class BaseClusterConfig(BaseEntityConfig, ABC):
    ...

    @property
    def network_cidr(self):
        return self.cluster_networks[0].cidr if self.cluster_networks else None

    @property
    def network_host_prefix(self):
        return self.cluster_networks[0].host_prefix if self.cluster_networks else None

    @property
    def service_network_cidr(self):
        return self.service_networks[0].cidr if self.service_networks else None

```